kaitmore/snippet
# data layer

Resolves #8

updates:
- add sqlite3 + sequelize
- add events, listeners, and app reducer

to test:
- `npm i && npm start`
- create a new snippet with ctrl+X
- update the snippet by clicking on the text (should transform into "this snippet has been updated!")
- delete snippets using the delete button
